themes/bootstrap3/templates/epf/home.phtml

```diff
@@ -0,0 +1,2 @@
+<?php
```

Comment:
This template could be slightly simplified to:

`<?=$this->render('search/home.phtml');`

Reply:
Done.